### PR TITLE
#168373102 Fix rating stats and readtime estimation

### PR DESCRIPTION
**Pull request by nkpremices**
##### What does this PR do?
* Have the `readtime` always rounded up
* Have a default object for article ratings stats

### CODE REVIEW

**Review by alainburindi — APPROVED** [review]

**Review by Ramadhan0 — APPROVED** [review]

File: src/helpers/calculate.ratings.percentages.js [hunk 1]
```diff
@@ -27,6 +29,26 @@ export default (data) => {
     percentage: percentages[data.indexOf(element)]
   }));
 
+  if (data.length < MAX_DATA_LENGTH) {
```
Comment (MemunaHaruna) [inline src/helpers/calculate.ratings.percentages.js:34]: @nkpremices a comment to describe what you're doing here would be helpful.

File: tests/article.ratings.test.js [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import chai from 'chai';
+import chai, { expect } from 'chai';
```
Comment (MemunaHaruna) [inline tests/article.ratings.test.js:1]: @nkpremices can you add a test for the read time estimation fix?

**Review by MemunaHaruna — COMMENTED** [review]
@nkpremices I left a few comments

File: src/helpers/calculate.ratings.percentages.js (hunk 1 above)
Comment (premlogic) [reply src/helpers/calculate.ratings.percentages.js:34]: Lemme do that @MemunaHaruna

File: tests/article.ratings.test.js (hunk 2 above)
Comment (premlogic) [reply tests/article.ratings.test.js:1]: I don't know how I can add a test for that as I'm just rounding up @MCFrank16 
Give me an idea please

Comment (premlogic) [reply tests/article.ratings.test.js:1]: But I'm going to try

**Review by MemunaHaruna — APPROVED** [review]

**Pull request merged**